client/src/components/user/UserRouter.js

```diff
@@ -15,7 +15,7 @@ const UserRouter = () => {
 
   //DEFINE HISTORY FUNCS AND PATH
   let history = useHistory();
-  const login = () => history.push(`/user/dashboard`);
+  const login = useCallback(() => history.push(`/user/dashboard`), [history]);
```

Comment:
Here I just made sure login is not gonna change between re-renders.
Anyways, I think I am gonna need your help on this one.
I am having hard times understanding this component.
Spent like 2 hours on this... There is something I am missing...